test/service/test_manifest.py

```diff
@@ -453,12 +449,17 @@ def test_manifest_zarr(self):
                     lines = response.content.decode().splitlines()
                     tsv_file = csv.DictReader(lines, delimiter='\t')
                     rows = list(tsv_file)
-                    rows = [dict(file_crc32c=row['file_crc32c'],
-                                 file_name=row['file_name'],
-                                 file_uuid=row['file_uuid'],
-                                 file_drs_uri=row['file_drs_uri'],
-                                 specimen_from_organism_organ=row['specimen_from_organism.organ']) for row in rows]
-                    self.assertEqual(expected, rows)
+                    actual = []
+                    for row in rows:
+                        drs_uri = furl(row['file_drs_uri'])
+                        self.assertTrue(drs_uri.origin.startswith('drs://drs'), drs_uri.origin)
+                        self.assertIsNotNone(drs_uri.host)
+                        actual.append(dict(file_crc32c=row['file_crc32c'],
```

Comment:
You should be able to stick with a comprehension if you write a nested function that transforms the actual value of `file_drs_uri` to the expected value AND asserts its other properties.